programs/drift/src/controller/isolated_position.rs

```diff
@@ -250,16 +250,23 @@ pub fn transfer_isolated_perp_position_deposit<'c: 'info, 'info>(
             .force_get_isolated_perp_position_mut(perp_market_index)?
             .get_isolated_token_amount(&spot_market)?;
 
+        // i64::MIN is used to transfer the entire isolated position deposit
+        let amount = if amount == i64::MIN {
```

Comment:
@crispheaney for my understanding on the client-side, does this mean we would pass in like -9223372036854775808? For the instruction?